pom.xml

```diff
@@ -67,11 +67,11 @@
     <mockwebserver.version>3.5.0</mockwebserver.version>
 
     <!-- Production -->
-    <cxf.version>3.3.2</cxf.version>
+    <cxf.version>3.3.5</cxf.version>
     <cxf.java2ws.plugin.version>${cxf.version}</cxf.java2ws.plugin.version>
-    <httpcomponents.httpcore.version>4.4.11</httpcomponents.httpcore.version>
+    <httpcomponents.httpcore.version>4.4.12</httpcomponents.httpcore.version>
```

Comment:
https://github.com/apache/brooklyn-server/pull/1096/files is showing 4.4.13

Reply:
4..4.12 is right - I looked in cxf 3.3.5 feature definition in an installed karaf with `feature:info`.